multimedia/whispercpp/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2024 Dengfeng Liu <liudf0716@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=whispercpp
+PKG_VERSION:=1.5.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ggerganov/whisper.cpp/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=06eed84de310fdf5408527e41e863ac3b80b8603576ba0521177464b1b341a3a
+PKG_BUILD_DIR:=$(BUILD_DIR)/whisper.cpp-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Dengfeng Liu <liudf0716@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/whispercpp
+  SECTION:=multimedia
+  CATEGORY:=Multimedia
+  DEPENDS:=+libstdcpp +ffmpeg
+  TITLE:=Port of OpenAI's Whisper model in C/C++
+  URL:=https://github.com/ggerganov/whisper.cpp
+endef
+
+define Package/whispercpp/description
+	Whisper is a general-purpose speech recognition model.
+	Whispercpp is a	port of OpenAI's Whisper model in C/C++.
```

Comment:
```suggestion
  Whisper is a general-purpose speech recognition model.
  Whispercpp is a port of OpenAI's Whisper model in C/C++.
```

Reply:
please use 2 `<space>` here

and the character between `a	port` should be `<space>` too

Reply:
please, do it again. this change seems to be rollbacked in latest commit.